ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-eus-4.10.yaml

```diff
@@ -40,43 +40,42 @@ releases:
   intermediate:
     release:
       architecture: amd64
-      channel: fast
+      channel: candidate
```

Comment:
Here the `candidate` channel means OCP official channel, right? If I remember right, the order is like `candidate` -> `fast` -> `stable`, right?

Reply:
Candidate channel is not something we recommend customers to use. It includes all the engineering candidate, release candidate and standard/GA releases.

> the order is like candidate -> fast -> stable

That's right